buildSrc/src/main/groovy/nva.identity.service.java-common-conventions.gradle

```diff
@@ -37,10 +37,6 @@ tasks.named('test') {
     finalizedBy jacocoTestReport
 }
 
-test {
```

Comment:
removed from conventions because they cause environment variable to be set in a global level and not have overview where and why we need them